decode.rb

```diff
@@ -0,0 +1,57 @@
+MORSE = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}.freeze
+
+def decode_char(char)
+  if MORSE.key? char
+    MORSE[char]
+  else
+    'Input is not morse code'
+  end
+end
+
+def dedcode_word(word)
```

Comment:
- [ ] Your app is throwing an error due to a mismatch in method definition and method call. Kindly ensure to improve upon this aspect

**Error:**
![image](https://user-images.githubusercontent.com/12745474/150415921-8c5e5b75-dc40-486f-bc56-f377888ee720.png)